repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/cors/CorsFilterBasicTest.groovy

```diff
@@ -42,80 +42,116 @@ class CorsFilterBasicTest extends ReposeValveTest {
         repose.waitForNon500FromUrl(reposeEndpoint)
     }
 
-    @Unroll("Non-CORS request with method OPTIONS and a CORS header request Method: #method")
-    def "OPTIONS request without origin header will bypass CORS and make to origin service"() {
-        def headers = [
-                (CorsHttpHeader.ACCESS_CONTROL_REQUEST_METHOD.toString()): method]
+    @Unroll
+    def "Non-CORS request with method OPTIONS, 'Access-Control-Request-Method' header of #method, and path #path should make it to origin service"() {
```

Comment:
Huh, it use to not interpolate in the test names, if you wanted the values in the test names you had to use unroll.